From d2d844414fc6570bed5402bd7785132ca1a6fb50 Mon Sep 17 00:00:00 2001
From: Carlos Garnacho <carlosg@gnome.org>
Date: Wed, 28 Jun 2017 23:41:21 +0200
Subject: [PATCH] build: Bump optional tracker dependency to 2.0

No code changes required. It remains API compatible with the current
usage in nautilus.
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index ef8293a53..85c3cb9da 100644
--- a/meson.build
+++ b/meson.build
@@ -82,7 +82,7 @@ if get_option ('enable-selinux')
 endif
 
 if get_option ('enable-tracker')
-    tracker_sparql = dependency ('tracker-sparql-1.0')
+    tracker_sparql = dependency ('tracker-sparql-2.0')
     conf.set10 ('ENABLE_TRACKER', true)
 endif
 
-- 
2.13.0

